docs/guides/opera_rtc_product_guide.md

```diff
@@ -0,0 +1,247 @@
+acquired and added to ASF's archive, the OPERA project continues to process them to RTC-S1 products.  
+
+RTC-S1 products generated by the OPERA project are all archived by ASF and can be accessed using a number of methods: 
+
+  - ASF's [Vertex Data Search](https://search.asf.alaska.edu/#/?dataset=OPERA-S1&productTypes=RTC "search.asf.alaska.edu" ){target=_blank}, 
+    which is a user-friendly and map-centric interface specialized for search and discover of ASF's SAR holdings
+  - Programmatically using the [asf_search python package](https://docs.asf.alaska.edu/asf_search/basics/ "docs.asf.alaska.edu" ){target=_blank}
+  - NASA's [Earthdata Search](https://search.earthdata.nasa.gov/search/granules?p=C2777436413-ASF "search.earthdata.nasa.gov" ){target=_blank} 
+    interface, which provides access to all of NASA's Earth Science datasets
+
+For more information on options for accessing archived OPERA RTC-S1 products, refer to ASF's 
+[OPERA Sentinel-1 RTC Tutorial](https://storymaps.arcgis.com/stories/dc2807b444924fc3a76c117a2c909f8b "OPERA Sentinel-1 RTC StoryMap Tutorial" ){target=_blank}. 
+For more information on the technical specifications of the OPERA RTC-S1 products, refer to JPL's 
+[RTC Product Documentation](https://www.jpl.nasa.gov/go/opera/products/rtc-product/ "OPERA RTC Product" ){target=_blank}.
+
+### OPERA RTC-S1 Products On Demand
+
+You can also order OPERA RTC-S1 products from ASF on demand. This is particularly useful if you need a time 
+series of RTC-S1 products that extends beyond the start of the archive. On-Demand processing is available for any 
+Sentinel-1 burst with the same burst ID (footprint) as an existing OPERA RTC-S1 product. 
+
+On-Demand RTC-S1 products are generated using the same code that is used by the OPERA project, but are processed 
+using ASF's 
+[HyP3](https://hyp3-docs.asf.alaska.edu/){target=_blank} 
+cloud-native processing platform instead of JPL's 
+[OPERA SDS](https://software.nasa.gov/software/NPO-52101-1){target=_blank} 
+processing management software. 
+
+Archived and On-Demand RTC-S1 products can be used interchangeably in a time series analysis.
+
+## Ordering On-Demand OPERA RTC-S1 Products
+
+
+
+
+
+## Product Packaging
+
+
+
+
+
+## On-Demand RTC Products from ASF
+
+In addition to OPERA RTC-S1 products, which use JPL's open-source 
+[ISCE3 software](https://github.com/isce-framework/isce3 "github.com/isce-framework/isce3" ){target=_blank} 
+to perform radiometric terrain correction, ASF also offers 
+[On-Demand RTC products](rtc_product_guide.md "Sentinel-1 RTC Product Guide" ){target=_blank} 
+generated using commercial [GAMMA](https://gamma-rs.ch/gamma-software "gamma-rs.ch/gamma-software" ){target=_blank} 
+SAR processing software. 
+
+These products are both high-quality Sentinel-1 RTC options, and you can use either with confidence for any given 
+RTC-based analysis workflow. Because the products do use different algorithms for the RTC processing, however, 
```

Comment:
```suggestion
RTC-based analysis workflow. Because OPERA products do use different algorithms for the RTC processing, however, 
```

Reply:
I'll give this one some thought. The "the" refers to both available products, but I can rework to be more explicit on the two products I'm discussing.

Reply:
@jacquelynsmale Does the wording change work for you?